Update README.md for CPUTemperature (#1122)
dotnet/iot: src/devices/CpuTemperature/README.md

## Summary

Returns the current temperature of the CPU Temperature Sensor. Useful telemetry in its own right, but also useful for calibrating the Raspberry Pi Sense HAT.
  

> Note: No hardware needs to connected for this to work. It gets the CPU core temperature, typically about 40 degress Celcius.  

Reviewer: ```suggestion
Note: No hardware needs to be connected in order for this binding to work. The way it works is that it gets the CPU core temperature, which is typically around 40 degrees Celsius.  
```

> On a Raspberry Pi. it does not work with Windows 10 IoT-Core; Raspian is the required operating system. 

Reviewer: ```suggestion
The binding requires a Raspberry Pi running Raspbian. It won't work if the Raspberry Pi's OS is Windows 10 IoT-Core. 
```

Reviewer 2: The binding requires any Linux (possibly Unix will work as well), Raspberry PI is not required.

> 
> Same as ```/opt/vc/bin/vcgencmd measure_temp```

Reviewer: ```suggestion
```
